static/app/components/events/interfaces/frame/contextLine.tsx

```diff
@@ -1,44 +1,116 @@
-import {Fragment} from 'react';
 import styled from '@emotion/styled';
 import classNames from 'classnames';
 
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+export type CoverageStatus = 'uncovered' | 'covered' | 'partial';
```

Comment:
Would it be better to have this in `static/app/types/integrations.tsx` with the rest of the codecov types?